test/e2e/manifest/manifest.go

```diff
@@ -105,9 +71,8 @@ func StatefulSetFromManifest(fileName, ns string) (*appsv1.StatefulSet, error) {
 	return &ss, nil
 }
 
-// DaemonSetFromManifest returns a DaemonSet from a manifest stored in fileName in the Namespace indicated by ns.
-func DaemonSetFromManifest(fileName, ns string) (*appsv1.DaemonSet, error) {
```

Comment:
also unused?

Reply:
yes . it is unused .

Reply:
Unused since the commit 38efff564c739d3e5d5aba90aca0498413a1907f